src/call_log/call_log.c

```diff
@@ -210,11 +210,34 @@ __wt_call_log_open_session(WT_SESSION_IMPL *session, int ret_val)
 
     WT_RET(__call_log_print_start(session, "connection", "open_session"));
     WT_RET(__call_log_print_input(session, 0));
-    WT_RET(__wt_snprintf(buf, sizeof(buf), "\"objectId\": \"%p\"", session));
+    WT_RET(__wt_snprintf(buf, sizeof(buf), "\"object_id\": \"%p\"", session));
     WT_RET(__call_log_print_output(session, 1, buf));
     WT_RET(__call_log_print_return(session, ret_val, ""));
 
     return (0);
 }
 
+/*
+ * __wt_call_log_set_timestamp --
+ *     Print the call log entry for the set timestamp API call.
+ */
+int
+__wt_call_log_set_timestamp(WT_SESSION_IMPL *session, const char *config, int ret_val)
+{
+    WT_CONNECTION_IMPL *conn;
+    char config_buf[128];
+    char objectid_buf[128];
+
+    conn = S2C(session);
+
+    WT_RET(__call_log_print_start(session, "connection", "set_timestamp"));
+    WT_RET(__wt_snprintf(objectid_buf, sizeof(objectid_buf), "\"object_id\": \"%p\"", conn));
```

Comment:
Why is object_id here? Shouldn't it be before [here](https://github.com/wiredtiger/wiredtiger/pull/8090/files#diff-da6411ea5bea33a18266c248bb258f7fff0f585b33ffcf16f9da19b060930cc6R237)?

Reply:
That object_id is an input in this case since we will use it to look up the session in the session map in the simulator.